docs: add sidecar container healthcheck (#2842)
aws/copilot-cli: site/content/docs/include/sidecar-config.en.md

The command to run to determine if the sidecar container is healthy.
The string array can start with `CMD` to execute the command arguments directly, or `CMD-SHELL` to run the command with the container's default shell.

<span class="parent-field">healthcheck.</span><a id="healthcheck-interval" href="#healthcheck-interval" class="field">`interval`</a> <span class="type">Duration</span>  
Time period between health checks, in seconds. Default is 10s.

<span class="parent-field">healthcheck.</span><a id="healthcheck-retries" href="#healthcheck-retries" class="field">`retries`</a> <span class="type">Integer</span>  
Number of times to retry before container is deemed unhealthy. Default is 2.

<span class="parent-field">healthcheck.</span><a id="healthcheck-timeout" href="#healthcheck-timeout" class="field">`timeout`</a> <span class="type">Duration</span>  
How long to wait before considering the health check failed, in seconds. Default is 5s.

<span class="parent-field">healthcheck.</span><a id="healthcheck-start-period" href="#healthcheck-start-period" class="field">`start_period`</a> <span class="type">Duration</span>  

> Grace period within which to provide containers time to bootstrap before failed health checks count towards the maximum number of retries. Default is 0s.

Reviewer: Here and in `site/content/docs/include/image-healthcheck`...something like:
```suggestion
Length of grace period for containers to bootstrap before failed health checks count towards the maximum number of retries. Default is 0s.
```
